composites/Plugin/Shared/components/YoastInput.js

```diff
@@ -18,7 +18,11 @@ export const YoastInputLabel = styled.label`
 `;
 
 export const YoastInputField = styled.input`
-	border: 1px solid ${ colors.$color_input_border };
-	padding: 0.75em;
-	font-size: 1em;
+	&&& {
+		box-shadow: inset 0 1px 2px rgba(0,0,0,.07);
```

Comment:
from the style-guide readme:
> do not use rgba colors in the palette, instead use the `rgba()` function

I'd agree the wording of this recommendation could be improved. However, it's meant to not introduce in the codebase colors that are not part of the palette. In this case it's a minor issue because it's black but for consistency I'd recommend to use `${ rgba( colors.$color_black, 0.7 ) }`